build/phpstan.neon

```diff
@@ -14,6 +14,7 @@ parameters:
 		- '#Constant PHPSTAN_TEST_CONSTANT not found#'
 		- '#Strict comparison using === between PhpParser\\Node\\Expr\\ArrayItem and null will always evaluate to false#'
 		- '#Dynamic call to static method PHPUnit\\Framework\\#'
+		- '#Call to function in_array\(\) requires parameter \#3 to be true#'
```

Comment:
Why? Which place causes this error?

Reply:
Oh, never mind, I just tried it out. Will fix strict-rules once this is merged! 👍